src/main/resources/fabric.mod.json

```diff
@@ -4,17 +4,18 @@
   "version": "${version}",
   "name": "Bow Infinity Fix",
   "description": "Removes need to have an arrow in your inventory to use the Infinity I enchant on your bow.",
-  "authors": ["Parker8283"],
+  "authors": ["Parker8283","MeeniMc"],
   "contact": {
     "homepage": "https://www.curseforge.com/minecraft/mc-mods/bow-infinity-fix",
     "sources": "https://github.com/Parker8283/BowInfinityFix",
     "issues": "https://github.com/Parker8283/BowInfinityFix/issues"
   },
   "license": "LGPLv3",
+  "icon": "assets/bif/bif-logo-light.png",
```

Comment:
The logos are quality programmer art :)

Real question, do I really _need_ logos? I've never used modmenu so I'm not sure what it looks like without it. Does the menu look ugly or something without a logo? I can't imagine I'm the only mod that is super small/insignificant, and if they essentially ~~require~~ "strongly recommend" logos, I personally think that is an oversight on their part. I honestly don't feel like this mod is worthy of a logo, it's too small, haha. Though I do appreciate the effort :)

Reply:
Mods with no logo all have the 'default' logo: a question mark on a gray background. The mod certainly works without a logo, I just though why not and used some mad powerpoint skills.

Reply:
In my personal opinion, that quirk of the modmenu should be addressed. There should be a way to not render anything if a mod doesn't want a logo, [kinda like what Forge does by default](https://user-images.githubusercontent.com/5672271/123340867-915f4680-d512-11eb-837c-fc8a6e09b67b.png). But I guess that's not something I can fix :)

In that case, I can stick with these. But inspired by the fact that both you and I seem to be masters of programmer art, I just commissioned my sibling to make a proper logo for the mod. So apologies if I wipe out your logos later. Nothing personal :), but my CF page has been lacking in the logo department as well (I just literally ripped a game asset because CF doesn't let you post a mod without a logo...)

Reply:
I have removed the logo as well from this pr, if you do finally want to use it, the material remains available in a separate branch that I have not PR'd here 
https://github.com/MeeniMc/BowInfinityFix/tree/pr/1.17

Feel free to pick it if you want.